### PR TITLE
add settings-screen

### LINKED ISSUE #198: Settings Page
A page that shows the settings

### PR DESCRIPTION
**Pull request by florian-sabonchi**
closes #198

### CODE REVIEW

**Comment by florian-sabonchi** [comment]
image

File: apps/tasks/lib/pages/settings_page.dart [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            Column(
+              children: [
+                ListTile(
+                  leading: const Icon(Icons.brightness_medium),
+                  title: Text(context.localization!.darkMode),
+                  trailing: Switch(value: isDarkTheme, onChanged: (bool value){
+                    setState(() {
+                      isDarkTheme = !isDarkTheme;
+                    });
+                  },),
```
Comment (DasProffi) [inline apps/tasks/lib/pages/settings_page.dart:34]: Add functionality with the `ThemeModel`

File: apps/tasks/lib/pages/settings_page.dart [hunk 2]
```diff
@@ -0,0 +1,66 @@
+
+import 'package:flutter/material.dart';
+import 'package:helpwave_localization/localization.dart';
+import 'package:helpwave_theme/constants.dart';
+
+class SettingsPage extends StatefulWidget{
+  const SettingsPage({super.key});
```
Comment (DasProffi) [inline apps/tasks/lib/pages/settings_page.dart:7]: See #217, should now be called `SettingsScreen`

Please also add a description like
```
/// Screen for settings and other app options
```

File: apps/tasks/lib/pages/settings_page.dart [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+                  },
+                ),
+                ListTile(
+                  leading: const Icon(Icons.info_outline),
+                  title: Text(context.localization!.licenses),
+                  trailing: const Icon(Icons.arrow_forward),
+                  onTap: () => {
+
+                  },
```
Comment (DasProffi) [inline apps/tasks/lib/pages/settings_page.dart:50]: add `showLicensePage` with appropriate parameters

File: apps/tasks/lib/pages/settings_page.dart [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    return Scaffold(
+      appBar: AppBar(
+        title: Text(context.localization!.settings),
+      ),
+      body: ListView(
+        children: [
+          Padding(padding: const EdgeInsets.symmetric(horizontal: paddingSmall), child:
+            Column(
+              children: [
+                ListTile(
```
Comment (DasProffi) [inline apps/tasks/lib/pages/settings_page.dart:27]: Dividers are still missing, maybe `ListTile.divideTiles()` can be used

**Comment by use-to** [comment]
@florian-sabonchi please rewrite your commit message: `chor` != `chore`

File: apps/tasks/lib/pages/settings_page.dart (hunk 4 above)
Comment (use-to) [reply apps/tasks/lib/pages/settings_page.dart]: I would like to use `ListView` here (for better optimizations).
```dart
ListView(
	children:
	ListTile.divideTiles(
    	context: context,
    	tiles: [ /* ... */ ]
  	).toList(),
),
```

**Comment by use-to** [comment]
divider length != Material divider length
settings

File: apps/tasks/lib/pages/settings_page.dart [hunk 5]
```diff
@@ -0,0 +1,92 @@
+import 'package:flutter/material.dart';
+import 'package:helpwave_localization/localization.dart';
+import 'package:helpwave_theme/constants.dart';
+import 'package:helpwave_theme/theme.dart';
+import 'package:provider/provider.dart';
+
+
+/// Screen for settings and other app options
+class SettingsPage extends StatefulWidget{
+  const SettingsPage({super.key});
```
Comment (DasProffi) [inline apps/tasks/lib/pages/settings_page.dart:10]: Should be called `SettingsScreen`, see #217

**Review by DasProffi — CHANGES_REQUESTED** [review]

**Review by use-to — CHANGES_REQUESTED** [review]
It's still `settings_page`
Please resolve all reviews

**Comment by DasProffi** [comment]
and move it to the new folder screens which is in the current main-branch

**Review by use-to — CHANGES_REQUESTED** [review]
Delete `apps/tasks/lib/screens/settings_page.dart` @florian-sabonchi 
This pr is taking a little bit too long.

**Review by use-to — APPROVED** [review]

**Pull request merged**